src/pp_25_guessing_game_two.py

```diff
@@ -0,0 +1,96 @@
+import random
+
+
+def input_validator(input_number: int) -> bool:
+    """
+    user guess validator
+
+    Args:
+        input_guess (int): user input integer
+
+    Returns:
+        bool: approval of the validation
+    """
+    if 1 <= input_number <= 9:
+        return True
+
+    return False
+
+
+def guess_the_number(input_number: int, guess_number: int) -> dict:
+    """
+    Args:
+        input_number (int): user input number
+        guess_number (int): by programe guess number
+
+    Returns:
+        dict: validation
+    """
+    if not input_validator(input_number):
+        print("You have to give me the number between 1 and 9 (included). Try again!")
+        return False
+
+    return evaluate_guess(input_number, guess_number)
+
+
+def evaluate_guess(user_input: int, guess_number: int) -> dict:
```

Comment:
We can remove this whole method as this logic is not the solution to the problem presented in the task. (see the commend below)